ruby-kafka.gemspec

```diff
@@ -25,6 +25,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_runtime_dependency 'gssapi', '>=1.2.0'
```

Comment:
nope. It doesn't require dev versions.. check this
https://github.com/zenchild/gssapi/blob/master/lib/gssapi/lib_gssapi_loader.rb
so whenever you are using kerberos, you already have these libs installed. And these libs are required/loaded only when gssapi is required.